chains/v5/chains.json

```diff
@@ -2999,6 +3023,10 @@
             {
                 "url": "wss://kusama.api.integritee.network",
                 "name": "Integritee node"
+            },
+            {
+                "url": "wss://integritee-rpc.dwellir.com",
```

Comment:
this node is unreachable, maybe use this one: wss://integritee-ksm-rpc.dwellir.com/

![image](https://user-images.githubusercontent.com/16337578/193293707-d0acc05b-d601-4cf8-ae2f-b86160ff9630.png)